README.md

```diff
@@ -91,6 +91,104 @@ Provisioner](https://www.packer.io/docs/provisioners/shell.html) runs the
 necessary configuration tasks.  Then, Packer creates an AMI from the instance
 and terminates the instance after the AMI is created.
 
+### Container Image Caching
+
+Optionally, some container images can be cached during the AMI build process in order to reduce the latency of the node getting to a `Ready` state when launched. 
+
+To turn on container image caching:
+
+```
+cache_container_images=true make 1.23
+```
+
+When container image caching is enabled, the following images are cached:
+ - 602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/eks/kube-proxy:<K8s-Version>-eksbuild.1
```

Comment:
@bwagner5 don't you need to know which build to use here? The current correct versions to use for EKS v1.21 to v1.24 is `eksbuild.2`.

Reply:
the caching logic uses the managed addons API (describe-addon-versions) to find the default and latest version. 

For kube-proxy on K8s 1.21 it uses:

```
                {
                    "addonVersion": "v1.21.14-eksbuild.2",
                    "architecture": [
                        "amd64",
                        "arm64"
                    ],
                    "compatibilities": [
                        {
                            "clusterVersion": "1.21",
                            "platformVersions": [
                                "*"
                            ],
                            "defaultVersion": false
                        }
                    ]
                },
...
                {
                    "addonVersion": "v1.21.2-eksbuild.2",
                    "architecture": [
                        "amd64",
                        "arm64"
                    ],
                    "compatibilities": [
                        {
                            "clusterVersion": "1.21",
                            "platformVersions": [
                                "*"
                            ],
                            "defaultVersion": true
                        }
                    ]
                },
```

Reply:
Ahhh, it might be clearer if you used the form `602401143452.dkr.ecr.<AWS_REGION>.amazonaws.com/eks/kube-proxy:<K8s-Version>-eksbuild.<BUILD_VERSION>` in the docs?